config/services.yaml

```
@@ -0,0 +1,11 @@
+services:
+    Oneup\ContaoSentryBundle\Twig\ContaoSentryTwigExtension:
```

As of current best practices we should not use FQCN as service ids.
```suggestion
    oneup_contao_sentry.twig.extension:
        class: Oneup\ContaoSentryBundle\Twig\ContaoSentryTwigExtension
```